### PR TITLE
 specs for control node

### PR DESCRIPTION
**Pull request by mrbojangles3**

### CODE REVIEW

**Comment by qmonnet** [comment]
Just a question: Why is it relevant to give the specs that we use internally?

For the context, I'm not against adding specs that may help users to understand what hardware is known to work. My concern is tying the project and docs to Hedgehog as a company. Could we maybe rephrase it as “The VLAB is known to work on the following setup:” or “The project CI runs on:”, or “Project maintainers use the following setup:”?

**Comment by Frostman** [comment]
@qmonnet the main goal is to have an easily referenceable section in docs to share with prospects as our tested recommendation.

**Review by Frostman — APPROVED** [review]

**Comment by qmonnet** [comment]
@Frostman Got it, but presented as Hedgehog 's (the company) setup, it doesn't give a vendor-neutral look to the project. Looking ahead, if we intend to set up some open-source governance model for the Fabric in the future, then there will be no reason that one vendor gets to list their setup in the project's docs, and not competition.

It's probably early to care too much about that, given at the moment it's necessary to register with Hedgehog to get the artefacts and run the Fabric anyway, but I think it's still something worth considering 🙂

**Comment by Frostman** [comment]
> @Frostman Got it, but presented as Hedgehog 's (the company) setup, it doesn't give a vendor-neutral look to the project. Looking ahead, if we intend to set up some open-source governance model for the Fabric in the future, then there will be no reason that one vendor gets to list their setup in the project's docs, and not competition.
> 
> It's probably early to care too much about that, given at the moment it's necessary to register with Hedgehog to get the artefacts and run the Fabric anyway, but I think it's still something worth considering 🙂

100% agree with you, at some point we will need to do a cleanup and separate company and project. Some of the examples would be docs under company domain, company name in the API namespace, Broadcom sonic dependency, etc.
